eventually/utils/validators.py

```diff
@@ -267,6 +268,31 @@ def event_data_validate(data, required_keys):
     is_data_valid = len(errors) == 0
     return is_data_valid
 
+def event_paginator_validate(request):
```

Comment:
This method should only validate the data! Does not have to return a new dictionary, it should return True or False.

Reply:
Done.